monkey/monkey_island/cc/environment/utils.py

```diff
@@ -44,8 +43,60 @@ def _create_secure_directory_windows(path: str):
             windows_permissions.get_security_descriptor_for_owner_only_perms()
         )
         win32file.CreateDirectory(path, security_attributes)
+
+    except Exception as ex:
+        LOG.error(f'Could not create a directory at "{path}": {str(ex)}')
+        raise ex
+
+
+def create_secure_file(path: str):
+    if not os.path.isfile(path):
+        if is_windows_os():
+            _create_secure_file_windows(path)
+        else:
+            _create_secure_file_linux(path)
+
+
+def _create_secure_file_linux(path: str):
+    try:
+        flags = (
+            os.O_RDWR | os.O_CREAT | os.O_EXCL
+        )  # read/write, create new, throw error if file exists
+        mode = 0o700  # read/write/execute permissions to owner
+        os.close(os.open(path, flags, mode))
+
     except Exception as ex:
-        LOG.error(
-            f'Could not create a directory at "{path}": {str(ex)}'
+        LOG.error(f'Could not create a file at "{path}": {str(ex)}')
+        raise ex
+
+
+def _create_secure_file_windows(path: str):
+    try:
+        file_access = win32file.GENERIC_READ | win32file.GENERIC_WRITE
+        file_sharing = (
+            win32file.FILE_SHARE_READ
+        )  # subsequent open operations on the object will succeed only if read access is requested
```

Comment:
Strange, why do we need `()` here?

Reply:
That's how black formats it. Because it's in the next line.

Reply:
That means the comment needs to be above the statement and the statement needs to be one like, like so: `file_sharing = win32file.FILE_SHARE_READ`